lib/sharing/utils.js

```diff
@@ -351,7 +351,7 @@ const getDescriptionFile = (fs, directory, descriptionFiles, callback) => {
 			);
 		}
 		const filePath = join(fs, directory, descriptionFiles[i]);
-		readJson(fs, filePath, (err, data) => {
+		const innerCallback = (err, data) => {
```

Comment:
I believe the main value of `readJson` being a separate function (even if it is optional in the `inputFileSystem` object) is that it allows that function to be cached independently of general file system reads. This is particularly relevant for `package.json` files, though arguably caching of `package.json` files could be handled in a separate layer.

Reply:
Yeah, you are right, let's postpone it, maybe for webpack v6